src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -96,6 +96,10 @@ const ReportActionItemFragment = (props) => {
             }
             const {html, text} = props.fragment;
 
+            if (props.fragment.isDeletedParentAction) {
```

Comment:
These lines introduced [a regression](https://github.com/Expensify/App/issues/24216), as in `ReportActionItem.js` we added a margin based on the report action's `isAttachment` property.